src/renderer/global.css

```diff
@@ -47,6 +47,22 @@
     --color-accent4: 240 95 168;
     --color-accent5: #8c9599;
     --color-accent6: #1db100;
+
+    --success-bg: hsl(143, 85%, 96%);
```

Comment:
2 suggestions, pick one of them :)

1. Add those to `tailwind.config.js`, you can follow the same approach as other colors such as `primary`, `secondary`, `destructive`, etc. This is better because the LSP gives you auto complete.
2. Just add tailwind colors to variant in `table.tsx` instead of using `hsl`